example.rules

```diff
@@ -0,0 +1,9 @@
+# The count of CPUs per node, useful for getting CPU time as a percent of total.
+instance:node_cpu:count = count(node_cpu{mode="idle"}) without (cpu,mode)
```

Comment:
Yes?  That's what is there.  In order to get the correct count, a random mode is selected.  I figured that was more efficient than doing a count of a count.